tests/e2e/support/api/davSpaces/spaces.ts

```diff
@@ -20,24 +23,20 @@ export const folderExists = async ({
   return getResponse.status === 200
 }
 
-export const createFolder = async ({
+const createFolder = async ({
   user,
-  folder
+  folder,
+  endPath
 }: {
   user: User
   folder: string
+  endPath: string
```

Comment:
what is an "endPath"? I wouldn't know what to give to this parameter

Reply:
i will give a suitable rename to it and also a nice comment.